### PR TITLE
Add myself to maintainers.

### PR DESCRIPTION
**Pull request by jezdez**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This just adds me as a maintainer of this feedstock due to me being a maintainer of Conda.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @isuruf @jakirkham @kalefranz @mbargull @mcg1969 @mingwandroid @msarahan @mwcraig @ocefpaf @patricksnape @pelson @scopatz
\ No newline at end of file
+* @isuruf @jakirkham @kalefranz @mbargull @mcg1969 @mingwandroid @msarahan @mwcraig @ocefpaf @patricksnape @pelson @scopatz @jezdez
```
Comment (jakirkham) [inline .github/CODEOWNERS:1]: JFYI this is handled through re-rendering

**Review by jakirkham — COMMENTED** [review]
Thanks Jannis! 😄

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render

**Review by jakirkham — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (jezdez) [reply .github/CODEOWNERS]: Ah, got it, thanks for letting me know!

**Comment by jakirkham** [comment]
@jezdez not sure if you saw, but details about joining conda-forge can be found here ( https://github.com/conda-forge/conda-feedstock/commit/bc3e8f37c4ada1ebf1d20a1a14edef6a24d3d837#commitcomment-52776948 )

**Comment by jezdez** [comment]
> @jezdez not sure if you saw, but details about joining conda-forge can be found here ( [bc3e8f3#commitcomment-52776948](https://github.com/conda-forge/conda-feedstock/commit/bc3e8f37c4ada1ebf1d20a1a14edef6a24d3d837#commitcomment-52776948) )

Yeah, that's super nice automation, I joined the chat room and will take a closer look at the maintenance docs to better understand how I can help in the future. Thanks!

**Comment by jakirkham** [comment]
Many years and lots of work by lots of people 😄

Great please let us know if you have any questions 🙂

We also have someone helping us with the docs. So please let us know if there are things we can improve there (always great to have a fresh set of eyes on them 😉)

**Pull request merged**